### PR TITLE
PC001 completed. Tests passed

### PR DESCRIPTION
**Pull request by nonickid**
ATTENTION: before clicking "Create Pull Request" please submit some meta data, thanks!

Difficulty level (1-10): [1]
Estimated time spent (hours): [1]
Completed (yes/no): [yes]

### CODE REVIEW

File: 01/nonickid/wordvalue.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    with open('../' + DICTIONARY) as fn:
+        return [word.strip() for word in fn.readlines()]
+
+def calc_word_value(word):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    return sum([ LETTER_SCORES.get(key.upper(), 0) for key in word])
```
Comment (bbelderbos) [inline 01/nonickid/wordvalue.py]: you can omit the list comprehension [] in `sum`

File: 01/nonickid/wordvalue.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+def max_word_value(words=None):
+    """Calculate the word with the max value, can receive a list
+    of words as arg, if none provided uses default DICTIONARY"""
+    if words is None:
+        words = load_words()
+    return max([ (word, calc_word_value(word)) for word in words ], key=lambda x: x[1])[0]
```
Comment (bbelderbos) [inline 01/nonickid/wordvalue.py]: I would use multiple lines here for readability

File: 01/nonickid/wordvalue.py (hunk 1 above)
Comment (bbelderbos) [reply 01/nonickid/wordvalue.py]: love `get`

**Comment by bbelderbos** [comment]
Great job @nonickid

File: 01/nonickid/wordvalue.py (hunk 1 above)
Comment (nonickid) [reply 01/nonickid/wordvalue.py]: Thank you for valuable comment

File: 01/nonickid/wordvalue.py (hunk 2 above)
Comment (nonickid) [reply 01/nonickid/wordvalue.py]: Thank you

**Pull request merged**